pyproject.toml

```diff
@@ -166,12 +166,10 @@ dependencies = [
 
 [project.optional-dependencies]
 llm = [
-    "transformers",
-    "torch",
     "langchain<=0.0.301", # Weirdly fails with newer version, see https://linear.app/giskard/issue/GSK-1853/updating-lanchain-version-over-00301-weirdly-fails
-    "evaluate",
-    "bert-score",
```

Comment:
both of `evaluate` and  `bert_score` are needed for control_chars_injection, why were they removed?